src/libs/ReportUtils.ts

```diff
@@ -4296,44 +4166,44 @@ function getIOUReportActionDisplayMessage(reportAction) {
     translationKey = ReportActionsUtils.isSplitBillAction(reportAction) ? 'iou.didSplitAmount' : 'iou.requestedAmount';
     return Localize.translateLocal(translationKey, {
         formattedAmount,
-        comment,
+        comment: transactionDetails?.comment ?? '',
     });
 }
 
+function isReportDraft(report: OnyxEntry<Report>): boolean {
+    return isExpenseReport(report) && report?.stateNum === CONST.REPORT.STATE_NUM.OPEN && report.statusNum === CONST.REPORT.STATUS.OPEN;
```

Comment:
```suggestion
    return isExpenseReport(report) && report.stateNum === CONST.REPORT.STATE_NUM.OPEN && report.statusNum === CONST.REPORT.STATUS.OPEN;
```

Reply:
@nkuoch we need to have optional chaining here because report could be null

Reply:
isExpenseReport is already making sure report is not null ... but I guess lint is complaining if you don't?

Reply:
Yeah , thats true